app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -176,11 +177,19 @@ protected void onStop() {
     @Override
     protected void onPause() {
         mAudioEngine.pauseEngine();
+        GeckoSession current = SessionStore.get().getCurrentSession();
```

Comment:
I think SessionStore.setActive() would be better since in the future we will probably have more than just one active GeckoSession. Also, this potentially won't work if there are multiple sessions on the store stack from a onCreateNewSession call.